### ISSUE TITLE
Autowire support

### ISSUE THREAD

**Issue by dkarlovi** [FriendsOfSymfony/FOSOAuthServerBundle#540]
With 1.5.x, services are not aliased properly for Symfony's autowire to work correctly:

```
  1x: Autowiring services based on the types they implement is deprecated since Symfony 3.3 and won't be supported in version 4.0. You should rename (or alias) the "fos_oauth_server.client_manager.default" service to "FOS\OAuthServerBundle\Model\ClientManagerInterface" instead.
    1x in ImageValidationServiceIntegrationTest::testCanValidateImageUsingSymfonyValidator from Tests\ImageBundle\Service
```

We should check what needs to get done for this to work correctly.

**Comment by dkarlovi** [comment]
@stof could you point me to the right direction, maybe?

**Comment by dkarlovi** [comment]
Example https://github.com/nelmio/alice/issues/711

**Comment by pfeilda** [comment]
That would be really nice I'm struggling at the same point. Maybe we can get a solution together.

**Comment by Ousret** [comment]
I fixed it by adding this to services.yml

```yml
  FOS\OAuthServerBundle\Model\AccessTokenManagerInterface:
    alias: fos_oauth_server.access_token_manager.default
```

**Comment by dkarlovi** [comment]
@Ousret yes, we just need to ship those with the bundle AFAIK.

**Comment by dkarlovi** [comment]
Closed with #560
